CHANGES.rst

```diff
@@ -4,6 +4,9 @@ Products.CMFCore Changelog
 3.0 (unreleased)
 ----------------
 
+- SkinsTool: fix Find form with Zope4 ZMI
```

Comment:
Actually, I would either not mention the Zope version, or call it "Zope 5", because a few lines below in the changelog we say: "Drop support for Zope 4.x", which makes the "Zope4" here a bit confusing. :-)

```suggestion
- SkinsTool: fix Find form with Zope ZMI
```

Reply:
Thanks, that's a good point.